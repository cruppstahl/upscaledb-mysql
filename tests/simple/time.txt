DROP TABLE IF EXISTS test;
CREATE TABLE test (t TIME) ENGINE=upscaledb;
INSERT INTO test VALUES ('13:01:28');
SELECT * FROM test;
INSERT INTO test VALUES ('13:01:29');
SELECT * FROM test;
INSERT INTO test VALUES ('13:01:30');
SELECT * FROM test;
