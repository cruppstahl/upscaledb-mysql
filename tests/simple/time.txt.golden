--------------
DROP TABLE IF EXISTS test
--------------

--------------
CREATE TABLE test (t TIME) ENGINE=upscaledb
--------------

--------------
INSERT INTO test VALUES ('13:01:28')
--------------

--------------
SELECT * FROM test
--------------

t
13:01:28
--------------
INSERT INTO test VALUES ('13:01:29')
--------------

--------------
SELECT * FROM test
--------------

t
13:01:28
13:01:29
--------------
INSERT INTO test VALUES ('13:01:30')
--------------

--------------
SELECT * FROM test
--------------

t
13:01:28
13:01:29
13:01:30
